.github/workflows/email-add-entry.yml

```
@@ -0,0 +1,135 @@
+          CITATION_KEY="${{ steps.lookup.outputs.citation_key }}"
+          READDATE="${{ github.event.client_payload.readDate }}"
+
+          # Append new entry to custom_info.yaml
+          cat >> contents/custom_info.yaml << 'EOF'
+
+${{ steps.lookup.outputs.citation_key }}:
+  readDate: '${{ github.event.client_payload.readDate }}'
+  akirahayashi_com:
+    tags: []
+    review: ''
+EOF
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)